### PR TITLE
[Tizen/Native] upload low light image enhancement pipeline app

### PR DESCRIPTION
**Pull request by niley7464**
upload the Tizen Native sample app with pipeline API

Signed-off-by: yelini-jeong <yelini.jeong@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

File: Tizen.native/LowLightImageEnhancementPipeline/src/main.c [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+  error_code = ml_pipeline_sink_register(pipeline_handle, "tensor_sink",
+                                         new_data_cb, NULL, &sink_handle);
+  if (error_code != ML_ERROR_NONE) {
+    dlog_print(DLOG_ERROR, LOG_TAG, "ml_pipeline_sink_register %d", error_code);
+  }
+
+  error_code = ml_pipeline_start(pipeline_handle);
+  if (error_code != ML_ERROR_NONE) {
+    dlog_print(DLOG_ERROR, LOG_TAG, "ml_pipeline_start %d", error_code);
+    ml_pipeline_sink_unregister(src_handle);
```
Comment (jaeyun-jung) [inline Tizen.native/LowLightImageEnhancementPipeline/src/main.c:249]: ml_pipeline_src_release_handle() to release src handle.

File: Tizen.native/LowLightImageEnhancementPipeline/src/main.c [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+    ml_pipeline_sink_unregister(sink_handle);
+    ml_pipeline_destroy(pipeline_handle);
```
Comment (jaeyun-jung) [inline Tizen.native/LowLightImageEnhancementPipeline/src/main.c:251]: or...just call ml_pipeline_destroy(pipeline_handle); and remove above 2 lines to release src/sink handle.
Calling ml_pipeline_destroy() internally releases element handles in the pipeline.

File: Tizen.native/LowLightImageEnhancementPipeline/src/main.c [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+
+/**
+ * @brief This callback function is called once after the main loop of the
+ * application exits.
+ *
+ * @param user_data The data passed from the callback registration function(not
+ * used here)
+ */
+static void app_terminate(void *data) { /* Release all resources. */
+}
```
Comment (jaeyun-jung) [inline Tizen.native/LowLightImageEnhancementPipeline/src/main.c:303]: recommend to release resource here.

ml_pipeline_destroy(pipeline_handle);
ml_tensors_info_destroy(info);
ml_tensors_info_destroy(input);

ecore_pipe_del(data_output_pipe);
image_util_decode_destroy(decode_h);

File: Tizen.native/LowLightImageEnhancementPipeline/src/main.c (hunk 2 above)
Comment (niley7464) [reply Tizen.native/LowLightImageEnhancementPipeline/src/main.c:249]: Removed 2 lines to release handles
Thank you

File: Tizen.native/LowLightImageEnhancementPipeline/src/main.c (hunk 3 above)
Comment (niley7464) [reply Tizen.native/LowLightImageEnhancementPipeline/src/main.c:307]: I added code for release resouce
Thank you 👍

**Review by jaeyun-jung — APPROVED** [review]
LGTM

**Pull request merged**